jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -114,43 +119,160 @@ public static boolean isCRIUSupportEnabled() {
 		return criuSupportEnabled;
 	}
 
+	private String imagesDir;
```

Comment:
you can set the field defaults here instead of in the constructor

Reply:
the defaults will also need to be documented. Perhaps documenting them in the constructor javadocs is best

Reply:
https://github.com/eclipse-openj9/openj9/pull/13185#discussion_r671345998